webapp-django/crashstats/urls.py

```diff
@@ -47,6 +47,10 @@
         'crashstats.profile.urls',
         namespace='profile'
     )),
+    (r'^help/', include(
```

Comment:
"help" is very different from the word "documentation". 
Help is when you're stuck or something.